packages/appstore/src/components/CFDs/cfd-demo-accounts.tsx

```diff
@@ -0,0 +1,181 @@
+import { DetailsOfEachMT5Loginid } from '@deriv/api-types';
+
+const CFDDemoAccounts = ({ isDerivedVisible, isFinancialVisible, current_list }: TCFDAccountsProps) => {
+    const available_demo_accounts: TStaticAccountProps[] = [
+        {
+            name: 'Derived',
+            description: localize('Trade CFDs on MT5 with Derived indices that simulate real-world market movements.'),
+            is_visible: isDerivedVisible(CFD_PLATFORMS.MT5),
+            disabled: false,
+            platform: CFD_PLATFORMS.MT5,
+            type: 'synthetic',
+        },
+        {
+            name: 'Financial',
+            description: localize('Trade CFDs on MT5 with forex, stocks & indices, commodities, and cryptocurrencies.'),
+            is_visible: isFinancialVisible(CFD_PLATFORMS.MT5),
+            disabled: false,
+            platform: CFD_PLATFORMS.MT5,
+            type: 'financial',
+        },
+        {
+            name: 'Deriv X',
+            is_visible: isDerivedVisible(CFD_PLATFORMS.DXTRADE),
+            description: localize(
+                'Trade CFDs on Deriv X with Derived indices, forex, stocks & indices, commodities and cryptocurrencies.'
+            ),
+            disabled: false,
+            platform: CFD_PLATFORMS.DXTRADE,
+            type: 'synthetic',
+            // ToDo: deriv x should have type of all in new API
+            // type: 'all'
+        },
+    ];
+
+    const { client, modules, common, ui }: TRootStore = useStores();
+    const {
+        standpoint,
+        dxtrade_tokens,
+        createCFDAccount,
+        setCurrentAccount,
+        setMT5TradeAccount,
+        toggleMT5TradeModal,
+        enableCFDPasswordModal,
+        openAccountNeededModal,
+        has_maltainvest_account,
+    } = modules.cfd;
+    const { platform, setAppstorePlatform } = common;
+    const { openTopUpModal } = ui;
+    const { is_eu } = client;
+
+    const openAccountTransfer = (
+        data: DetailsOfEachMT5Loginid & { account_id?: string; platform?: string },
+        meta: { category: string; type?: string }
+    ) => {
+        setCurrentAccount(data, meta);
+        openTopUpModal();
+    };
+
+    const REAL_DXTRADE_URL = 'https://dx.deriv.com';
+    const DEMO_DXTRADE_URL = 'https://dx-demo.deriv.com';
+
+    const getDXTradeWebTerminalLink = (category: string, token?: string) => {
+        let url = category === 'real' ? REAL_DXTRADE_URL : DEMO_DXTRADE_URL;
+
+        if (token) {
+            url += `?token=${token}`;
+        }
+
+        return url;
+    };
+
+    const openCFDAccount = (account_type: string) => {
+        if (is_eu && !has_maltainvest_account && standpoint.iom) {
+            openAccountNeededModal('maltainvest', localize('Deriv Multipliers'), localize('demo CFDs'));
+        } else {
+            createCFDAccount({
+                category: 'demo',
+                type: account_type,
+                platform,
+            });
+            enableCFDPasswordModal();
+        }
```

Comment:
```suggestion
        if (is_eu && !has_maltainvest_account && standpoint.iom) {
            openAccountNeededModal('maltainvest', localize('Deriv Multipliers'), localize('demo CFDs'));
            return;
        }
        
        createCFDAccount({
            category: 'demo',
            type: account_type,
            platform,
        });
        enableCFDPasswordModal();
```
You can remove the `else` block to make it more readable